packages/genesys-spark-components/src/components/stable/gux-time-picker/gux-time-picker.tsx

```diff
@@ -78,6 +83,26 @@ export class GuxTimePicker {
   @Listen('blur')
   onBlur() {
     if (this.valueLastChange !== this.value) {
+      // Format input time to match format found in the popup time list (e.g. "01:30" -> "1:30", "00:30" -> "12:30", etc)
```

Comment:
Do you all think it makes sense to trigger input validation on blur or instead on input value change? I was worried triggering validation on input value change would be more intrusive than on blur.